### DISCUSSION TITLE
Welcome to Spectober_Fest Discussions!

### DISCUSSION THREAD

**Discussion by debidattasuryaprakash** [Spectrum-CETB/Spectober_Fest#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

### Please go through the [Code of Conduct](https://github.com/Spectrum-CETB/Spectober_Fest/blob/main/CODE_OF_CONDUCT.md) for Contributors.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Aditya-150** [comment]
### All contributors give a wave 👋 to it.
